src/wp-content/themes/twentyfifteen/inc/customizer.php

```diff
@@ -792,10 +793,23 @@ function twentyfifteen_color_scheme_css_template() {
 		'secondary_sidebar_textcolor' => '{{ data.secondary_sidebar_textcolor }}',
 		'meta_box_background_color'   => '{{ data.meta_box_background_color }}',
 	);
-	?>
-	<script type="text/html" id="tmpl-twentyfifteen-color-scheme">
-		<?php echo twentyfifteen_get_color_scheme_css( $colors ); ?>
-	</script>
-	<?php
+
+	$css = twentyfifteen_get_color_scheme_css( $colors );
+
+	if ( function_exists( 'wp_print_inline_script_tag' ) ) {
+		wp_print_inline_script_tag(
+			$css . "\n//# sourceURL=" . __FUNCTION__,
+			array(
+				'type' => 'text/html',
+				'id'   => 'tmpl-twentyfifteen-color-scheme',
+			)
+		);
+	} else {
+		printf(
+			"<script %s>\n%s\n\t</script>\n",
+			'type="text/html" id="tmpl-twentyfifteen-color-scheme"',
+			$css
+		);
```

Comment:
This seems like it could be simplified:

```suggestion
		echo "<script type="text/html" id="tmpl-twentyfifteen-color-scheme">$css</script>";
```